src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -32,6 +32,11 @@ class HoldConfirmation extends React.Component {
    */
   requireUser() {
     if (this.state.patron && this.state.patron.id) {
+      const queryID = this.props.location.query ? this.props.location.query.id : '';
```

Comment:
I tried between nypltest1 and my personal account locally and it seems not to work for me. Any better way I can test it?